core/EE_Registry.core.php

```diff
@@ -1467,7 +1471,9 @@ public function getAddon($class_name)
     public function removeAddon($class_name)
     {
         $class_name = str_replace('\\', '_', $class_name);
-        unset($this->addons->{$class_name});
+        if (isset($this->addons->{$class_name})) {
+            unset($this->addons->{$class_name});
```

Comment:
isset checks are not needed before unsetting something. It's completely safe to just unset.

Reply:
:+1: